modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -474,6 +488,89 @@ You can test whether the service is available at `_API_PLATFORM_HOST_/apiplatfor
 |===
 
 
+[[encrypt_password]]
+== Encrypt Passwords in an Existing mule-agent.yml File
+
+
+If you didn't set the `AGENT_VAR_master_password` environment variable when registering the agent 
+with Runtime Manager, you can encrypt these passwords later in the `mule-agent.yml` file:
+
+`keystorePassword`::
+Keystore password used to access the `mule-agent.jks` file
+`keystoreAliasPassword`:: 
+Keystore alias password used to access the `mule-agent.jks` file
+`password`::
+Password used for authenticating with a proxy server
+
+=== Prerequisites 
+
+* Runtime Manager agent version 2.4.17
++
+To update previous versions of the agent, see <<Update the Agent>>. 
+
+=== Encrypt Passwords
+
+To encrypt passwords in your `mule-agent.yml` file:
+
+. Set the `AGENT_VAR_master_password` environment variable to the master password.
++
+`export AGENT_VAR_master_password=myPassword`
+. Run the encryption utility:
++
+`$MULE_HOME/bin/amc_setup --encrypt`
++
+The encryption utility replaces the passwords in the `keystorePassword`, `keystoreAliasPassword`, and `password` fields in the `mule-agent.yml` file with the encrypted passwords, for example:
++
+[source,yaml,linenums]
+----
+globalConfiguration:
+  security:
+    keyStorePassword: '![PBEWITHSHA1ANDDESEDE,Fe4kl2UBnGRoyh7H2v9lq+QSG7Pe6ZP8k34xmZEHmcAb1jABAwPLk2xWrGtSt1iE]'
+    keyStoreAlias: agent
+    keyStoreAliasPassword: '![PBEWITHSHA1ANDDESEDE,EtqRDPq7UTxDHzZGpOGazfcveEi832Pqm1mqhlwYRGLZGh9NSNXie6nAZr09b3iT]'
+  authenticationProxy:
+    endpoint: https://data-authenticator.anypoint.mulesoft.com:443
+  proxyConfiguration: 
+    password: '![PBEWITHSHA1ANDDESEDE,VIs82yFDp66lnsL/Rss8q8js/zPDQ8+HJpdNepdZtA9obeDj5u+z4HIocsbLvXow]'
+----
+
+The encrypted passwords start with *!* and are enclosed in square brackets *[ ]*.
+
+=== Update and Reencrypt Passwords
+
+To update an encrypted password in the `mule-agent.yml` file: 
+
+. Edit `mule-agent.yml`.
+. Replace the encrypted password with the new plain-text password.
+. Set the `AGENT_VAR_master_password` environment variable: 
++
+`export AGENT_VAR_master_password=myPassword`
+. Run the encryption utility:
++
+`$MULE_HOME/bin/amc_setup --encrypt`
+
+=== Display Passwords in mule-agent.yml in Plain Text 
+
+To display the passwords in `mule-agent.yml` in plain text, run the decryption utility: 
+
+`$MULE_HOME/bin/amc_setup --decrypt`
+
+This command outputs the contents of the `mule-agent.yml` file with plain-text passwords.
+
+=== Update the Master Password
+
+To update the master password: 
+
+. Run the decryption utility:
```

Comment:
I'd like to ask your opinion about this: do you find it odd to have the stem sentence? Would you rather go straight to the first step from the title?